app/api/events.py

```diff
@@ -857,3 +857,38 @@ def clear_export_urls(event):
     event.xcal_url = None
     event.pentabarf_url = None
     save_to_db(event)
+
+
+class UpcomingEventList(EventList):
+    """
+    List Upcoming Events
+    """
+
+    def before_get(self, args, kwargs):
+        """
+        method for assigning schema based on admin access
+        :param args:
+        :param kwargs:
+        :return:
+        """
+        super().before_get(args, kwargs)
+        self.schema.self_view_many = 'v1.upcoming_event_list'
+
+    def query(self, view_kwargs):
+        """
+        query method for upcoming events list
+        :param view_kwargs:
+        :return:
+        """
+        current_time = datetime.now(pytz.utc)
+        query_ = self.session.query(Event).filter(
```

Comment:
This does not filter deleted events. Maybe that is done automatically in the library, but can you please still verify that deleted events are not being returned?

Reply:
Yes, deleted events are not returned.